webapp/components/interop-data.js

```diff
@@ -579,5 +579,408 @@ export const interopData = {
         'countsTowardScore': true
       }
     }
+  },
+  '2024': {
+    'table_sections': [
+      {
+        'name': 'Active Focus Areas',
+        'rows': [
+          'interop-2024-accessibility',
+          'interop-2024-starting-style-transition-behavior',
+          'interop-2023-property',
+          'interop-2024-dsd',
+          'interop-2024-dir',
+          'interop-2024-font-size-adjust',
+          'interop-2024-websockets',
+          'interop-2024-indexeddb',
+          'interop-2024-layout',
+          'interop-2024-nesting',
+          'interop-2023-events',
+          'interop-2024-popover',
+          'interop-2024-relative-color',
+          'interop-2024-video-rvfc',
+          'interop-2024-scrollbar',
+          'interop-2024-text-wrap',
+          'interop-2023-url',
+        ],
+        'score_as_group': false
+      },
+      {
+        'name': 'Active Investigations',
+        'rows': [
+          'Accessibility Testing',
+          'Mobile Testing',
+          'WebAssembly Testing',
+        ],
+        'score_as_group': true
+      },
+      {
+        'name': 'Previous Focus Areas',
+        'rows': [
+          'interop-2023-cssborderimage',
+          'interop-2023-color',
+          'interop-2023-container',
+          'interop-2023-contain',
+          'interop-2023-mathfunctions',
+          'interop-2023-pseudos',
+          'interop-2023-fonts',
+          'interop-2023-forms',
+          'interop-2023-has',
+          'interop-2023-inert',
+          'interop-2023-cssmasking',
+          'interop-2023-mediaqueries',
+          'interop-2023-modules',
+          'interop-2023-motion',
+          'interop-2023-offscreencanvas',
+          'interop-2022-scrolling',
+          'interop-2021-transforms',
+          'interop-2023-webcodecs',
+          'interop-2023-webcompat',
+          'interop-2021-aspect-ratio',
+          'interop-2022-cascade',
+          'interop-2022-dialog',
+          'interop-2021-position-sticky',
+          'interop-2022-text',
+          'interop-2022-viewport',
+          'interop-2022-webcompat'
+        ],
+        'score_as_group': false
+      },
+    ],
+    'investigation_scores': [
+      {
+        'name': 'Accessibility Testing',
+        'url': 'https://github.com/web-platform-tests/interop-2023-accessibility-testing',
+        'scores_over_time': []
+      },
+      {
+        'name': 'Mobile Testing',
+        'url': 'https://github.com/web-platform-tests/interop-2023-mobile-testing',
+        'scores_over_time': []
+      },
+      {
+        'name': 'WebAssembly Testing',
+        'url': '',
+        'scores_over_time': []
+      },
+    ],
+    'investigation_weight': 0.0,
```

Comment:
Can we start omitting this? It was only for Interop 2022. Perhaps the code that uses `investigation_weight` can just tolerate it being missing and not mix in scores?

Reply:
I can submit a separate PR to remove this from non-relevant years 🙂